tests/acceptance/learning/test_fault_tolerance.py

```diff
@@ -73,95 +73,131 @@ def bad_bytestring_of_known_nodes():
     assert "Failed to verify MetadataResponse from Teacher" in warning  # TODO: Cleanup logging templates
 
 
-@pytest.mark.skip("See Issue #1075")  # TODO: Issue #1075
-def test_invalid_operators_tolerance(testerchain,
-                                     test_registry,
-                                     blockchain_ursulas,
-                                     agency,
-                                     idle_staker,
-                                     application_economics,
-                                     ursula_decentralized_test_config
-                                     ):
+def test_invalid_operators_tolerance(
+    testerchain,
+    test_registry,
+    blockchain_ursulas,
+    threshold_staking,
+    agency,
+    application_economics,
+    ursula_decentralized_test_config,
+    mocker,
+):
```

Comment:
Good to see this test restored, even though it's outside the original intention of this PR.